clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -105,6 +109,9 @@
     private final Map<String, ?> configs;
     private final KafkaPrincipalBuilder principalBuilder;
     private final Map<String, AuthenticateCallbackHandler> callbackHandlers;
+    private final Map<String, Long> connectionsMaxReauthMsByMechanism;
+    private Long sessionExpirationTimeNanos;
+    private boolean connectedClientSupportsReauthentication = false;
```

Comment:
Initiaize in the constructor similar to other fields?

Reply:
👍 fixed